### DISCUSSION TITLE
Zotero not allow annotations?

### DISCUSSION THREAD

**Discussion by mike091020** [BryanJenksCommunity/FAQ#64] [Q&A] [answered]
Zotero does not allow me to extract notes made to web pages. I can get the web page with a browser plug-in and I can annotate with Foxit, but I can't extract the annotations.  I get an error. Anyone know a tool that will allow this.  I don't use PDFs all that often.  I mostly get my information from web pages.  Thanks for any suggestions.

**Comment by tallguyjenks** [comment, marked as answer]
the way to extract the markdown notes is done through the mdnotes extension which parses PDF's to pull out annotations. there are web extensions to pull out plain text, I think roam highlighter is one such extension, you ould try those for when you view the archived HTML doc in your browser. that or https://www.worldbrain.io
